packages/templates/clients/websocket/python/test/example.py

```
@@ -1,4 +1,4 @@
-from temp.snapshotTestResult.client_postman import PostmanEchoWebSocketClientClient
+from temp.snapshotTestResult.client_postman.client import PostmanEchoWebSocketClientClient
```

Ruff (0.8.2)

`temp.snapshotTestResult.client_postman.client.PostmanEchoWebSocketClientClient` imported but unused

Remove unused import: `temp.snapshotTestResult.client_postman.client.PostmanEchoWebSocketClientClient`

(F401)